Readability.js

```diff
@@ -1678,15 +1678,18 @@ Readability.prototype = {
    *
    * @return boolean Whether or not we suspect parse() will suceeed at returning an article object.
    */
-  isProbablyReaderable: function() {
-    var nodes = this._doc.getElementsByTagName("p");
-    if (nodes.length < 5) {
-      return false;
-    }
-
-    var possibleParagraphs = 0;
+  isProbablyReaderable: function(helperIsVisible) {
```

Comment:
I mean, we can write a fallback relying on `clientHeight`, but that requires not just a DOM implementation but a layout implementation, which implies a CSS implementation, which 'even' `jsdom` doesn't have, so I'm not sure how useful it is in the meantime. I guess it might be useful for the iOS version?